recorder.js

```diff
@@ -17,6 +17,7 @@ var Recorder = function( config ){
   this.config.encoderSampleRate = config.encoderSampleRate || 48000;
   this.config.encoderPath = config.encoderPath || 'oggopusEncoder.js';
   this.config.stream = config.stream || false;
+  this.config.reuseStream = !!config.reuseStream;
```

Comment:
Could we use the convention other options use `config.leaveStreamOpen || false` I would also say `leaveStreamOpen` is more accurate than reuse stream.